sdk/go/pulumi-language-go/main.go

```diff
@@ -198,10 +199,21 @@ func (host *goLanguageHost) GetRequiredPlugins(ctx context.Context,
 		return nil, err
 	}
 
+	args := []string{"list", "-m", "-json", "-mod=mod", "all"}
+
+	tracingSpan, _ := opentracing.StartSpanFromContext(ctx,
+		fmt.Sprintf("%s %s", gobin, strings.Join(args, " ")),
+		opentracing.Tag{Key: "component", Value: "exec.Command"},
+		opentracing.Tag{Key: "command", Value: gobin},
+		opentracing.Tag{Key: "args", Value: args})
```

Comment:
Possibly for later - but I wonder if we should have a generic wrapper over `exec.Command` that does this tracing - and use that everywhere we do process spawns?

Reply:
I like that idea https://github.com/pulumi/pulumi/issues/7311 put it in the backlog for the moment.